pkgs/applications/networking/cluster/tanka/default.nix

```diff
@@ -0,0 +1,23 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+
+buildGoModule rec {
+  pname = "tanka";
+  version = "0.11.1";
+
+  src = fetchFromGitHub {
+    owner = "grafana";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "0hp10qgalglsdhh6z6v4azh2hsr89mdrv1g5lssfl5jyink409yd";
+  };
+
+  vendorSha256 = "15x8fqz2d2793ivgxpd9jyr34njzi1xpyxdlfyj1b01n2vr3xg4m";
+
```

Comment:
:raising_hand: Could we please set the package version `main.Version=${version}` via `buildFlagsArray`.

As an example the `gopass` package:
 https://github.com/NixOS/nixpkgs/blob/e3cba5aad991a0f355545fc0b069cbf987b577d0/pkgs/tools/security/gopass/default.nix#L25

Reply:
Done !

Reply:
Looking at the tanka's [Makefile](https://github.com/grafana/tanka/blob/e5556f1c851bcfe2f0ec885896642d7c418c692f/Makefile#L17), the `buildFlagsArray` should be closer to `-ldflags=-s -w -X main.Version=${version}'`

Reply:
Effectivelly ! I check on the code source and `main.commit` didn't exist.

Reply:
@timstott Done.

Reply:
@mikefaille it should be `main.Version` (capital **V**). You should be able to test that the output from `tk --version` includes 0.11.1

Reply:
🤦 I just fixed and tested it ! Ty for your reviews !